pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -685,6 +690,24 @@ def do_ready(self):
             state = self.get_state()
             return state == TestState.RUNNING or state == TestState.STOPPED
 
+    def do_clear_env(self):
+        with self.lock:
+            try:
+                env_file_path = self.home_path('.env')
+                open(env_file_path, 'w').close()
+                print(f'File {env_file_path} has been successfully cleared.')
+            except Exception as e:
+                print(f'An error occurred while clearing the env: {e}')
+
+    def do_add_env(self, key_value):
+        with self.lock:
+            env_file_path = self.home_path('.env')
+            # Create the file if it does not exit
+            open(env_file_path, 'a').close()
```

Comment:
is this necessary? Just testing it out locally, it seems that the file doesnt need to exist beforehand right?

Reply:
Yes. In the docs it says that `dotenv.set_key(env_file_path, key, value)` will throw if the file does not exist.

Reply:
hmm yea i see that. that's weird, because just testing it out, it def doesnt crash, and looking at the code, it does a `pathlib.Path(path).touch()` on the filename, but I agree with just following what the docs say here anyway